README.md

```diff
@@ -138,9 +138,12 @@ app = modal.App(name="golinks")
 vol = modal.Volume.from_name("golinks-data", create_if_missing=True)
 
 image = modal.Image.from_registry(
-    "golang:1.23.0-bookworm",
+    "golang:1.23.4-bookworm",
     add_python="3.10",
-).run_commands(["go install -v github.com/tailscale/golink/cmd/golink@latest"])
+).run_commands([
+    "go install -v github.com/tailscale/golink/cmd/golink@latest",
+    "rm -rf /root/.config"
```

Comment:
/root/.config/golinks.db is removed by this command if I understand correctly. I believe this volume was added intentionally.

Reply:
Runs before volume is mounted. If you don’t do this there is some go telemetry crap in the folder and modal cannot mount the storage volume into the mountpath.

Modal is weird.

Reply:
If it's just the go telemetry data, I believe there is an env var we can set to turn that off